sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/FileIndexSuite.scala

```diff
@@ -248,6 +248,25 @@ class FileIndexSuite extends SharedSQLContext {
       assert(spark.read.parquet(path.getAbsolutePath).schema.exists(_.name == colToUnescape))
     }
   }
+
+  test("SPARK-25062 - InMemoryCache stores only simple BlockLocations") {
```

Comment:
`InMemoryCache` -> `InMemoryFileIndex`? And, `simple BlockLocations` may look unclear later.